extensions/localserver-cli-command/src/compilers/Compiler.js

```diff
@@ -46,6 +46,11 @@ module.exports = class Compiler {
 
 	_createRequireJSFile(localFolder) {
 		const src = require.resolve('requirejs');
-		fs.copyFileSync(src, path.join(localFolder, 'require.js'));
+		const dst = path.join(localFolder, 'require.js');
+		fs.copyFileSync(src, dst);
+
+		let content = fs.readFileSync(dst).toString();
+		content = content.replace('#!/usr/bin/env node', '');
+		fs.writeFileSync(dst, content);
```

Comment:
Instead copying the file from src to dst, then reading it from dst, remove the `#!/usr/bin/env node` string and then write again in dst it, wouldn't it be more efficient to read it from src, remove the unwanted string and write it in dst?

Reply:
+1